resources/python/build_for_ultimaker.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+#
+# Copyright (C) 2019 Ultimaker B.V.
+#
+
+set -eu
+
+ARCH="armhf"
```

Comment:
Cura runs on x64 architectures. I think this is specific to certain projects and should probably just be committed to those projects rather than this one, or at least marked as such.